price_pusher/src/index.ts

```diff
@@ -79,13 +87,45 @@ if (CONTRACT_ADDR[argv.pythContract] !== undefined) {
 
 const priceConfigs = readPriceConfigFile(argv.priceConfigFile);
 
-async function run() {
+async function injectiveRun() {
```

Comment:
minor: I think `injective` and `evm` can have the same run function (or use a common function for running controller, ...) as many things are similar in your code.

Reply:
as mentioned above
will clean this file in separate PR

Reply:
FYI if you want to do this stuff in a separate PR, i suggest leaving a code comment like 

`// FIXME: unify with evmRun() below once the interfaces are figured out.`

then people will understand what you're trying to do when reading the PR and won't leave comments about it